apiary-metastore-events/sns-metastore-events/apiary-metastore-consumers/privileges-grantor/privileges-grantor-lambda/src/test/java/com/expediagroup/apiary/extensions/events/metastore/consumer/privilegesgrantor/lambda/PrivilegesGrantorLambdaTest.java

```diff
@@ -147,6 +163,20 @@ public void clientErrorOnFirstRecordsSecondRecordOk() throws Exception {
     assertThat(response.getDescription(), containsString("Privileges granted successfully"));
   }
 
+  private static final String BASE_EVENT_FROM_SNS = "{"
+      + "  \"Type\" : \"Notification\","
+      + "  \"MessageId\" : \"message-id\","
+      + "  \"TopicArn\" : \"arn:aws:sns:us-west-2:sns-topic\","
+      + "  \"Timestamp\" : \"2018-10-23T13:01:54.507Z\","
+      + "  \"SignatureVersion\" : \"1\","
+      + "  \"Signature\" : \"signature\","
+      + "  \"SigningCertURL\" : \"https://sns.us-west-2.amazonaws.com/SimpleNotificationService-xxxx\","
+      + "  \"UnsubscribeURL\" : \"https://sns.us-west-2.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-west-2:440407435941:sns-topic\",";
```

Comment:
What's this number `440407435941`?

Reply:
I copied this example from receiver code. It must be the AWS account id. I have removed it from both places.